src/View/ViewBuilder.php

```diff
@@ -526,6 +526,7 @@ public function build(
         ];
         $data += $this->_options;
 
+        /** @var \Cake\View\View */
```

Comment:
I didnt know this is possible or necessary :)
:+1:

Reply:
Leaving out the variable name is possible https://psalm.dev/docs/supported_annotations/#off-label-usage-of-the-var-tag